enhancement! : remove all field label, field hint, field errors from all possible inputs (#1693)
strapi/design-system: docs/stories/Inputs/Checkbox.mdx

Reviewer: i'd also get rid of:

```
#### Checked
Checked state is used to show that an option has been selected.
#### Indeterminate
Indeterminate state is used to show partially checked states.
#### Unchecked
Unchecked state is used to show that an option has been selected.
```

import { Meta, Canvas, ArgTypes } from '@storybook/blocks';
import { Checkbox } from '@strapi/design-system';

import * as CheckboxStories from './Checkbox.stories';

<Meta of={CheckboxStories} />

# Checkbox

- [Overview](#overview)
- [Usage](#usage)
- [Props](#props)
- [Variants](#variants)
- [Accessibility](#accessibility)

## Overview

Checkboxes allow users to choose one or multiple values from a range of options.
[View source](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/Checkbox)

<Canvas of={CheckboxStories.Base} />

## Usage

```js
import { Checkbox } from '@strapi/design-system';
```

- Use the [Radio](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/Radio) button component if the user should select only one value.
- Checkbox labels are aligned differently based on language direction: to the right in LTR languages and to the left in RTL languages.

## Props

<ArgTypes of={Checkbox} />

## Variants

### Basic checkbox

<Canvas of={CheckboxStories.Base} />

> ### checkbox Group

Reviewer: ```suggestion
### Checkbox Group
```

# Checkbox

- [Overview](#overview)
- [Usage](#usage)
- [Props](#props)
- [Variants](#variants)
- [Accessibility](#accessibility)

## Overview

Checkboxes allow users to choose one or multiple values from a range of options.
[View source](https://github.com/strapi/design-system/tree/main/packages/strapi-design-system/src/Checkbox)

<Canvas of={CheckboxStories.Base} />

## Usage

```js
import { Checkbox } from '@strapi/design-system';
```

> - Use the Radio button component if the user should select only one value.

Reviewer: Let's add a link to the `Radio` component 👍🏼

### Disabled

> Checkboxes can possibly not be available to selection yet. This state can be related to a lack of permission or a need

Reviewer: ```suggestion
Checkboxes may not be available to select yet. This state can be related to a lack of permission or a need
```

to pursue another action first.

<Canvas of={CheckboxStories.Disabled} />

### With field

> Checkbox field with hint or error.

Reviewer: ```suggestion
Checkbox Field
```
